.taskcluster.yml

```diff
@@ -236,6 +236,8 @@ tasks:
         then:
           $let:
             version: {$eval: 'head_branch[10:]'}
+            linux_artifact: rust-code-analysis-linux-x86_64.tar.gz
+            windows_artifact: rust-code-analysis-windows-x86_64.zip
```

Comment:
Since you're here, let's also call this artifact `rust-code-analysis-win-x86_64.zip` (this naming is used more commonly). You'll need to update the `build_windows_release` task too.